### ISSUE TITLE
History: Remember sync settings

### ISSUE THREAD

**Issue by fhemberger** [tegon/traktflix#58]
When I check/uncheck episodes for syncing history with trakt, the plugin should remember those decisions. E.g. there are movies I only watched a couple of minutes before deciding that they're not my cup of tea, still Netflix considers them "watched".

Each time I open the history sync, those movies are marked for syncing again, even if I manually disabled syncing for them the last time.

**Comment by MrMamen** [comment]
I agree with the suggestion, but in your case it could help to delete them from netflix from here: https://www.netflix.com/viewingactivity

**Comment by tegon** [comment]
I agree with you, the problem is that I don't have any kind of storage for this extension right now.
Only the one that Chrome provides, which is not a lot of space, so I use to store the access token to query the Trakt API.
I want to give more configuration options to users, but to do that I would need to build some kind of backend.
Can you remove them from Netflix history directly like @MrMamen suggested above for now?

**Comment by fhemberger** [comment]
I did and it worked for me. Still would be nice as enhancement feature request. ;)
